interfaces/acados_matlab_octave/AcadosOcpSolver.m

```diff
@@ -31,51 +31,116 @@
 
 classdef AcadosOcpSolver < handle
 
-    properties
-        t_ocp % templated solver
+    properties (Access = public)
         ocp % Matlab class AcadosOcp describing the OCP formulation
-        qp_gettable_fields = {'qp_Q', 'qp_R', 'qp_S', 'qp_q', 'qp_r', 'qp_A', 'qp_B', 'qp_b', 'qp_C', 'qp_D', 'qp_lg', 'qp_ug', 'qp_lbx', 'qp_ubx', 'qp_lbu', 'qp_ubu', 'qp_zl', 'qp_zu', 'qp_Zl', 'qp_Zu'}
     end % properties
 
+    properties (Access = private)
+        fields = {'x', 'u', 'z', 'sl', 'su', 'lam', 'pi'};
+        qp_gettable_fields = {'qp_Q', 'qp_R', 'qp_S', 'qp_q', 'qp_r', 'qp_A', 'qp_B', 'qp_b', 'qp_C', 'qp_D', 'qp_lg', 'qp_ug', 'qp_lbx', 'qp_ubx', 'qp_lbu', 'qp_ubu', 'qp_zl', 'qp_zu', 'qp_Zl', 'qp_Zu'}
+        t_ocp % templated solver
 
+        % required info loaded from json
+        N
```

Comment:
N_horizon ?

Reply:
Addressed in https://github.com/acados/acados/pull/1472/commits/8945621d1018207ad08657491b59423e8662e96a